game_engine.properties

```diff
@@ -4,15 +4,3 @@
 ## first.
 
 engine_version = 1.9.0.0
-
-## URL where we will download the list of available maps
-map_list_file = https://raw.githubusercontent.com/triplea-game/triplea/master/triplea_maps.yaml?raw=true
-## Example that would use a local file config file, useful for map building or development
-# map_list_file = triplea_maps.yaml
-
-
-## Location where we download latest lobby description file, this tells us where the latest lobby is located
-lobby_properties_file_url = https://raw.githubusercontent.com/triplea-game/triplea/master/lobby_server.yaml?raw=true
```

Comment:
Thanks for the clarification.

> game_engine.properties is though not a bad answer for props that need to be enabled on startup.

The JFX client definitely falls into this category.

I think the serialization proxy stuff, at least for save games, can be moved, as you suggested, to the beta settings.  That's actually an improvement from the users' perspective, IMO.  I'll look into doing that once this PR is merged.